public/docs/_examples/forms/dart/web/main.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:angular2/bootstrap.dart';
```

Comment:
Yeah, we prefer using them in TypeScript, but it's only a matter of style. No pressure to use them in Dart. Especially not on `bootstrap.dart`, which exports only one function `bootstrap`.